test/e2e-tests.sh

```diff
@@ -23,15 +23,19 @@ failed=0
 # Run serverless-operator specific tests.
 (( !failed )) && serverless_operator_e2e_tests || failed=2
 if [[ $TEST_KNATIVE_KAFKA == true ]]; then
-  (( !failed )) && install_strimzi || failed=6
-  (( !failed )) && serverless_operator_kafka_e2e_tests || failed=7
+  (( !failed )) && install_strimzi || failed=3
+  (( !failed )) && serverless_operator_kafka_e2e_tests || failed=4
 fi
 
-(( !failed )) && ensure_serverless_installed || failed=3
+(( !failed )) && ensure_serverless_installed || failed=5
```

Comment:
This will need the INSTALL_KNATIVE_KAFKA flag set to true to actually install it, so I guess we need to keep the flag in the Makefile.